tests/topotests/lib/topogen.py

```diff
@@ -81,35 +81,30 @@ def is_string(value):
 
 
 def get_exabgp_cmd(commander=None):
-    """Return the command to use for ExaBGP version < 4."""
+    """Return the command to use for ExaBGP."""
 
     if commander is None:
         commander = Commander("exabgp", logger=logging.getLogger("exabgp"))
 
     def exacmd_version_ok(exacmd):
-        logger.debug("checking %s for exabgp < version 4", exacmd)
+        logger.debug("checking %s for exabgp", exacmd)
```

Comment:
Maybe we should do a reverse check? To check if we don't use a lower version.

Reply:
done